components/batch-system/src/config.rs

```diff
@@ -8,8 +8,7 @@ use tikv_util::config::ReadableDuration;
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
-    #[online_config(skip)]
-    pub max_batch_size: Option<usize>,
+    pub max_batch_size: usize,
```

Comment:
This change may not be needed after https://github.com/tikv/tikv/pull/12133 is merged